terraform/database_dev/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "cf_api_url" {
```

Comment:
[suggestion/blocking] i think we can definitely take this out, as it's the same in the `shared` variable file.

[question] actually, can we take out all the variable blocks here? i think we just need the definitions of the variables if they are different than what's in the shared file. 

IE:
`"env" = "dev"`

Reply:
`env = dev` would be super cool, and take away my feelings of ickiness at basically having to duplicate the variables file.  I will do some experimentation!